apps/src/gamelab/animationListModule.js

```diff
@@ -300,17 +353,28 @@ export function addAnimation(key, props) {
   // TODO: Validate that key is not already in use?
   // TODO: Validate props format?
   return (dispatch, getState) => {
-    dispatch({
-      type: ADD_ANIMATION,
-      key,
-      props
-    });
+    props.looping = true;
```

Comment:
Our action creator probably shouldn't be mutating the input arguments. Might be better to instead just do:
`dispatch(addAnimationAction(key, { ...props, looping: true }))`.
Also, should the above two TODOs be addressed?